### DISCUSSION TITLE
Merge exclude option values

### DISCUSSION THREAD

**Discussion by konpeki622** [vitest-dev/vitest#701] [General]
According to the document, the default value of `exclude` option is `['**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**']`.

That sounds great. But if I set some excluded path like `['packages/template/*']`, test files in `node_modules` will be run, that means default value was ineffective and I should append them.

Maybe the option values could be auto-merged?

**Comment by sheremet-va** [comment]
That means you won't be able to override them tho which you might want to. Maybe we can export them, so you would just `...inline` them?

In fact, we can export the whole default config.

**Reply by konpeki622** [reply]
That's a solution🤔

**Reply by sheremet-va** [reply]
You can now import `configDefaults` from `vitest` in [0.3.1](https://github.com/vitest-dev/vitest/releases/tag/v0.3.1)

**Reply by konpeki622** [reply]
Thank you! I tried to import it with version 0.3.1 but the project threw an error when I run build: `No "exports" main defined in ...\node_modules\vitest\package.json`. Seems to be related to the Node version? My Node version is 16.13 👀

**Reply by konpeki622** [reply]
my mistake...it works
